lib/dk/local.rb

```diff
@@ -5,6 +5,8 @@ module Dk::Local
 
   class BaseCmd
 
+    attr_reader :scmd
+
```

Comment:
@jcredding ok good.  yeah, I was torn which way to go on this.  Ultimately, the flexibility of exposing `scmd` and the fact that this kept the remote cmd and local cmd APIs as similar as possible won out for me.